library/pod-security-policy/fsgroup/template.yaml

```diff
@@ -15,15 +17,23 @@ spec:
           type: object
           properties:
             rule:
+              description: "An FSGroup rule name.  Example: MayRunAs."
+              enum:
+                - MayRunAs
```

Comment:
When we are parsing schemas, we can probably list allowed enum values.

Reply:
removed `. Example: MayRunAs.`